src/index.tsx

```diff
@@ -14,6 +17,8 @@ ReactDOM.render(
   document.getElementById("root"),
 );
 
+newRemoteFileApiConnection();
```

Comment:
My understanding at a glance is that as soon as the player loads in the game will be listening for a remote connection on a particular port. Shouldn't the default be for there to be no remote connection, until/unless the player enables a remote connection setting in the options?

Reply:
![2022-08-11-184323_909x458_scrot](https://user-images.githubusercontent.com/19653491/184188181-486a3642-241a-4ebd-bae7-79f79b5df4fb.png)